docs/hugo/content/guide/asov1-asov2-migration/_index.md

```diff
@@ -0,0 +1,331 @@
+
+```
+kubectl api-resources --api-group cert-manager.io`
+```
+should show `v1` resources.
+
+> ℹ️ **Note:** We strongly recommend ensuring that you're running the latest version of cert-manager 
+> (1.14.4 at the time this article was written).
+
+> ℹ️ **Note:** We also recommend ensuring that ASOv1 is configured to use `cert-manager.io/v1` resources.
+> You can run `helm upgrade` and pass `--set certManagerResourcesAPIVersion=cert-manager.io/v1` to ensure ASOv1 is using
+> the v1 versions of the cert-manager resources.
+
+### Install ASOv2
+
+Follow the [standard instructions](../../#installation). We recommend you use the same credentials as ASOv1 is currently using.
+
+> ℹ️ **Note:** Make sure to follow the [guidelines](../crd-management/) for setting the `--crdPattern`. Configure ASOv2 
+> to only manage the CRDs which you need.
+> 
+> For example: if you're using storage accounts, resource groups, redis cache's, cosmosdbs, eventhubs, and SQL Azure, 
+> then you'd configure 
+> `--set crdPattern='resources.azure.com/*;cache.azure.com/*;documentdb.azure.com/*;eventhub.azure.com/*;sql.azure.com/*;storage.azure.com/*'`
+
+### Stop ASOv1 reconciliation
+
+Mark each ASOv1 resource with the `skipreconcile=true` annotation.
+This annotation ensures that ASOv1 can no longer update or delete the resource in question in Azure.
+
+To annotate a specific resource:
+```
+kubectl annotate -n <ns> storageaccount <name> skipreconcile=true
+```
+
+To annotate all ASOv1 resources in a given namespace:
+```
+kubectl annotate $(kubectl api-resources -o name | grep azure.microsoft.com | paste -sd "," - | xargs kubectl get -A -o name) -n ns1 skipreconcile=true`
+```
+
+Once you've annotated the resources, double-check that they all have the `skipreconcile` annotation. 
+
+> ⚠️ **Important**: If you delete an ASOv1 resource that does not have this annotation it will delete the underlying resource 
+> in Azure and may cause downtime or an outage
+
+### Use `asoctl` to import the resources from Azure into ASOv2
+
+Download the latest version of [asoctl](../../tools/asoctl/).
+
+Export the resources you want to transfer using the `asoctl import azure-resource` command. `import azure-resource`
+imports the referenced resource and all its child resources by default. This means if you want to import the resources 
+from a specific resource group being managed by ASOv1 you can just refer to the resource group.
+
+When running this command it's recommended you use at least asoctl v2.7.0 and pass the 
+`-n` (namespace) and `-a` (annotation) arguments to import the resources to a particular namespace with 
+the `serviceoperator.azure.com/reconcile-policy: skip` annotation.
+```
+asoctl import azure-resource /subscriptions/<subid>/resourceGroups/<rg> -o resources.yaml -n ns1 -a serviceoperator.azure.com/reconcile-policy=skip
+```
+
+This should produce a file that looks something like this (this is a simplified file showing just a resource group and
```

Comment:
Suggested rephrase to avoid sequential _this_

```suggestion
This should produce a file similar to the one shown here (this is a simplified file showing just a resource group and
```